pyanaconda/kickstart.py

```diff
@@ -601,15 +605,15 @@ def _check_boot_drive(self, storage, boot_drive, usable_disks):
 
         if not matched_disks:
             raise KickstartParseError(_("No match found for given boot drive "
-                                        "\"{}\".").format(boot_drive), lineno=self.lineno)
+                                        "\"{}\".").format(boot_drive))
 
         if len(matched_disks) > 1:
             raise KickstartParseError(_("More than one match found for given boot drive "
-                                        "\"{}\".").format(boot_drive), lineno=self.lineno)
+                                        "\"{}\".").format(boot_drive))
 
         if matched_disks[0] not in usable_disks:
             raise KickstartParseError(_("Requested boot drive \"{}\" doesn't exist or cannot "
-                                        "be used.").format(boot_drive), lineno=self.lineno)
+                                        "be used.").format(boot_drive))
```

Comment:
Yes, we loose the line number in a kickstart file. However, the boot drive can be set also from UI and from DBus, so the line number doesn't really make sense here.

Reply:
Not sure. If you have really big kickstart file and in that kickstart file this command is written multiple times it could be hard to debug.
Is there a way how to leave this information only for kickstart installations?

Reply:
Ideally, this check should be done sooner, probably in the `parse` method of the `bootloader` command. However, it might cause some new problems, since some of the storage commands are parsed in the DBus module and some in UI. Would it be ok to just create an issue and implement it later?

Reply:
I'm fine with that. However, it would be great to implement this before it will land to rhel-8.